doc/reference.md

```diff
@@ -355,6 +356,23 @@ of the host array types.
 | `array32 len format`   | `Array32 len (Repr format)`         |
 | `array64 len format`   | `Array64 len (Repr format)`         |
 
+### Repeat formats
+
+The `repeat_until_end` format repeats parsing the given format until the end of
+the current binary stream is reached:
+
+- `repeat_until_end : Format -> Format`
+
+#### Representation of repeat formats
+
+Because the repeat format does not have a predefined length, it is
+[represented](#format-representations) as a dynamically sized
+[array type](#array-types):
```

Comment:
Is it possible to access the length once parsed? Do we need to mention that the repeat format doesn't need to be of fixed size?

Reply:
Yeah, I think I'll need to get up to adding sizing to this reference, like how I talk about the host representation types.

Are you thinking the byte length? Or the number of elements in the array?

Reply:
I was thinking of number of elements in the array, as the byte length could be determined by following it with a pos.